src/config.ts

```diff
@@ -0,0 +1,23 @@
+type Config = {
+  apiKey?: string;
+  authDomain?: string;
```

Comment:
what is this?

Reply:
i dont have it in my local .env and it's still fine

Reply:
idk just part of the firebase config